lib/src/main/java/com/auth0/msg/DataLocation.java

```diff
@@ -0,0 +1,5 @@
+package com.auth0.msg;
+
+public enum DataLocation {
+    FRAGMENT, QUERYPART
```

Comment:
please change QUERYPART to  QUERY_STRING (more explicit). Also add one more value FORM_POST